src/ducks/SANCharts/ChartPage.js

```diff
@@ -376,13 +362,6 @@ class ChartPage extends Component {
                       />
                     </div>
                   )}
-
-                  <div className={styles.moreData}>
-                    <span className={styles.limitedLabel}>
-                      See much more data in our
-                    </span>
-                    <Link to='/dashboards'>SANbase Dashboards</Link>
```

Comment:
Removed link to dashboards (used at /projects/bitcoin page) :( I hope I know about it.

Reply:
Can you share a screenshot/link where it's in Figma?

Reply:
@DmitriVanGuard , here (bottom)
https://www.figma.com/file/RPLINi2nrD30hQjcANws3CxH/Sanbase?node-id=8082%3A5128

But I placed here old component(screenshot) because for new Link and upgradeBtn need to change styles of ChartWidget.js(metrics, remove old upgradeBtn in right side) - it is for another PR
  
![image](https://user-images.githubusercontent.com/14061779/63634027-a8ecf000-c659-11e9-94fa-391d48f58e45.png)

But, if this component(screenshot) is not important you can remove it :)

Reply:
`See much more data in our SANbase Dashboards` is a part of the old implementation and is not present in the up-to-date figma design (`master`)